src/mode/mode_config_data.js

```diff
@@ -96,6 +97,19 @@ cwc.mode.Mod.prototype.getMod = function(helper) {
  */
 cwc.mode.ConfigData = {};
 
+
+/**
+ * AIY mode.
+ */
+cwc.mode.ConfigData[cwc.mode.Type.AIY] = new cwc.mode.Mod({
+  authors: ['Filip Stanis'],
+  mime_types: [cwc.utils.mime.Type.PYTHON.type],
```

Comment:
Please check mime_types to `cwc.utils.mime.Type.CWC.type`.
`cwc.utils.mime.Type.PYTHON.type` would mean that you are open raw .py files with the AIY mode, but it should be more the case to open only AIY .cwc files with the AIY mode.

Reply:
This breaks layout right now. Any thoughts why?

Reply:
CWC will check automatically for supported modes based on the mime type, which mean if you define your mode with python you telling CWC to open all .py files in this mod as default which is wrong. That's the reason why it worked in your instance but you are disabling the standard python mode. Which mean you are no longer be able to use pure python without the aiy mode. That's the reason why you need to switch to the .cwc file to avoid such conflicts.

You also need to create the corresponding .cwc file under static_files/resources/templates and defining the correct mode.